var/spack/repos/builtin/packages/xalt/package.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Xalt(AutotoolsPackage):
+    """XALT 2 is a tool to allow a site to track user executables and library usage on a cluster.
+    When installed it can tell a site what are the top executables by Node-Hours or by the number
+    of users or the number of times it is run. XALT 2 also tracks library usage as well. XALT 2 can
+    also track package use by R, MATLAB or Python. It tracks both MPI and non-MPI programs."""
+
+    homepage = "https://xalt.readthedocs.io/en/latest/"
+    git = "https://github.com/xalt/xalt.git"
+
+    version("2.10.45", tag="xalt-2.10.45")
+
+    depends_on("py-mysqlclient", when="+MySQL")
+
+    # Multiple values variants
+    variant(
+        "config",
+        default="Config/rtm_config.py",
+        description="A python file defining the accept, ignore, hostname pattern lists",
+    )
+    variant(
+        "etcDir",
+        default=".",
+        description="Directory where xalt_db.conf and reverseMapD can be found",
+    )
+    variant(
+        "mode",
+        default="700",
+        values=("755", "750", "700"),
+        description="Override executable install mode (755 or 750 or 700)",
+    )
+    variant(
+        "primeNumber",
+        default="997",
+        description="The prime number of directories when using XALT_FILE_PREFIX",
+    )
+    variant(
+        "syshostConfig",
+        default="mapping:mapping.example.json",
+        description="How to determine syshost",
+    )
+    variant(
+        "tmpdir", default="/dev/shm", description="The tmp directory to use with package records"
```

Comment:
Not everyone will be able to write to `/dev/shm`.  Why not use a subdirectory of `self.stage` for portability?